app/src/main/java/com/peacecorps/pcsa/circle_of_trust/MessageDialogBox.java

```diff
@@ -20,52 +30,70 @@
  */
 public class MessageDialogBox extends DialogFragment {
 
-    private CircleOfTrustFragment objCircleOfTrustFragment;
+    private static CircleOfTrustFragment objCircleOfTrustFragment;
+    private static Context context;
+    private Dialog listDialog;
 
     //Need a compulsory empty constructor for recreation of dialog while handling config changes
     public MessageDialogBox()
     {
 
     }
-    public MessageDialogBox(CircleOfTrustFragment objCircleOfTrustFragment)
+
+    /**
+     * Checks whether the device is connected to a mobile network or not
```

Comment:
@Buddhiprabha Line 44 needs to be changed, I'm sorry about this. I shall make the necessary ammendments